ads/_config.js

```diff
@@ -1189,7 +1189,11 @@ const adConfig = jsonConfiguration({
   'sekindo': {
     renderStartImplemented: true,
   },
-
+  'sevio': {
```

Comment:
Nit: extra newline before and after.

Reply:
Added the newline before and after. Pls check. Thanks